packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | the `id` of the Role to identify the Binding                          | URL          | String         | TRUE     |
+
+#### Response parameters
+
+A paginated list of [Members](#member-object).
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings/cr0WxyzJxGIr0ouum0g4/members"
+```
+
+#### Response example
+
+```json
+{
+  "members": [
+    {
+      "id": "irb1qe6PGuMc7Oh8N0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/users/00uuk41Hjga5qGfQ30g3"
+        }
+      }
+    }, {
+      "id": "irb1q92TFAHzySt3x0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "binding": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/${resource-set-id}/bindings/${role-id-or-name}"
+    },
+    "next": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/${resource-set-id}/bindings/${role-id-or-name}/members?after=${last-member-id}"
+    }
+  }
+}
+```
+### Get a member from a binding
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}/members/${memberId}" />
+
+Get a Member of a Role in a Resource Set
+
+#### Request parameters
+
+| Parameter      | Description                                                           | Param Type   | DataType       | Required |
+| :------------- | :-------------------------------------------------------------------- | :----------- | :------------- | :------- |
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | `id` of the Role to identify the Binding                              | URL          | String         | TRUE     |
+| memberId       | `id` of the Member within the Binding                                 | URL          | String         | TRUE     |
+
+
+`memberId` is the id obtained when [listing Members within Binding](#list-members-in-a-binding). For example, if the Member object was:
+```json
+{
+  "id": "irb1qe6PGuMc7Oh8N0g4",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00uuk41Hjga5qGfQ30g3"
+    }
+  }
+}
+```
+`irb1qe6PGuMc7Oh8N0g4` could be used as `memberId` to remove the User from list of Members in the Binding.
+
+#### Response parameters
+
+Response is the id representing the Binding and a link to the actual Member object which could be a User or a Group.
```

Comment:
Response is the id representing the... > The ID that represents the...
Add a comma before "which"
Remove period since it's going to be a phrase, not a sentence.